data-preprocessing/JSON  Section/main.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+import csv
+
+# absolute location of the json files
+PATH = "."
+
+# search for json files in the directory and add their names to a list
+def put_json_filenames_in_list():
+    json_files = [] 
+    for filename in os.listdir(PATH):
+        if filename.endswith(".json"):
+            json_files.append(filename)
+    return json_files
+
+# load all json file
+def load_json_file(json_file):
+    with open(PATH+json_file, "r") as file:
+        entire_json = json.load(file)
+        return entire_json[0]
+
+# obtain the section from json file
+def Section(json_dictionary):
+    try:
+        section=json_dictionary["Section"]
+        return section
+    except:
+        return ''
+
+
+
+ALL_JSON_FILES = put_json_filenames_in_list()
+
+
+for file in ALL_JSON_FILES:
+    PAGES = []
+    SECTIONS = []
```

Comment:
since you are using this globally, would be good to initialize at the top of the scripts where you initialized `PATH`. So group all the constants there so it's clear